src/compiler/commandLineParser.ts

```diff
@@ -147,8 +147,8 @@ export const inverseJsxOptionMap: Map<string, string> = new Map(mapIterator(jsxO
 //       augmented in another lib.
 // NOTE: We must reevaluate the target for upcoming features when each successive TC39 edition is ratified in
 //       June of each year. This includes changes to `LanguageFeatureMinimumTarget`, `ScriptTarget`,
-//       `ScriptTargetFeatures` transformers/esnext.ts, compiler/commandLineParser.ts and the contents of each
-//       lib/esnext.*.d.ts file.
+//       `ScriptTargetFeatures` transformers/esnext.ts, compiler/commandLineParser.ts,
+//       compiler/utilitiesPublic.ts, and the contents of each lib/esnext.*.d.ts file.

```

Comment:
@billyjanitsch Sorry for the lack of information 🙇 

NOTE comments exist in multiple files and I would like to have them updated as well:

* `src/compiler/types.ts`
* `src/compiler/utilities.ts`
* `src/compiler/transformers/esnext.ts`

Also, please add this comment to `src/compiler/utilitiesPublic.ts`.